grails-app/domain/org/pih/warehouse/product/ProductSupplier.groovy

```diff
@@ -113,25 +115,28 @@ class ProductSupplier implements Serializable, Comparable<ProductSupplier> {
     }
 
     static PROPERTIES = [
-            "id"                           : "id",
-            "code"                         : "code",
-            "productCode"                  : "product.productCode",
-            "legacyProductCode"            : "productCode",
-            "productName"                  : "name",
-            "description"                  : "description",
-            "supplierId"                   : "supplier.id",
-            "supplierName"                 : "supplier.name",
-            "supplierCode"                 : "supplierCode",
-            "supplierProductName"          : "supplierName",
-            "manufacturerId"               : "manufacturer.id",
-            "manufacturerName"             : "manufacturer.name",
-            "manufacturerCode"             : "manufacturerCode",
-            "manufacturerProductName"      : "manufacturerName",
-            "defaultProductPackageUomCode" : "defaultProductPackage.uom.code",
-            "defaultProductPackageQuantity": "defaultProductPackage.quantity",
-            "defaultProductPackagePrice"   : "defaultProductPackage.price",
-            "standardLeadTimeDays"         : "standardLeadTimeDays",
-            "ratingTypeCode"               : "ratingTypeCode",
-            "comments"                     : "comments"
+            "ID"                                  : "id",
+            "Product Source Code"                 : "code",
+            "Product Source Name"                 : "name",
+            "Product Code"                        : "product.productCode",
+            "Product Name"                        : "product.name",
+            "Legacy Product Code"                 : "productCode",
+            "Supplier Name"                       : "supplier.name",
+            "Supplier Item No"                    : "supplier.code",
+            "Manufacturer Name"                   : "manufacturer.name",
+            "Manufacturer Item No"                : "manufacturer.code",
+            "Minimum Order Quantity"              : "minOrderQuantity",
+            "Contract Price (Each)"               : "contractPrice.currency.code",
+            "Contract Price Valid Until"          : ["property": "contractPrice.toDate", "dateFormat": "MM/dd/yyyy"],
+            "Rating Type"                         : "ratingTypeCode",
+            "Default Global Preference Type"      : "globalProductSupplierPreference.id",
```

Comment:
@pmuchowski Should not here be the descriptive value instead of ID? Something like: `globalProductSupplierPreference.prefetenceType.name`, `globalProductSupplierPreference.prefetenceType.validationCode` or combination of both (eg: `name (validationCode)`)?

Reply:
I wasn't sure about this one, I will change it to globalProductSupplierPreference.prefetenceType.name